File: AndroidStudy/mentoring/ClickerGame/app/src/main/java/com/example/clickergame/MainActivity.java
package com.example.clickergame;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    int count=0;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button plusButton,minusButton;
        TextView textView;
        plusButton=findViewById(R.id.plus_button);
        minusButton=findViewById(R.id.minus_button);
        textView=findViewById(R.id.text_view);
        textView.setText(String.valueOf(count));
        View.OnClickListener onClickListener= new View.OnClickListener(){

            @Override
            public void onClick(View v) {
                if(v.getId()==R.id.plus_button){
                    count++;
                }else if(v.getId()==R.id.minus_button){
                    count--;
                }
                textView.setText(String.valueOf(count));
            }
        };
        plusButton.setOnClickListener(onClickListener);
        minusButton.setOnClickListener(onClickListener);
    }
}

File: AndroidStudy/library/rx/rxAndroid/rxAndroidPractice/app/src/main/java/com/test/rxandroidpractice/MainActivity.kt
package com.test.rxandroidpractice

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.test.rxandroidpractice.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var mainActivityBinding:ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mainActivityBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(mainActivityBinding.root)
    }
}